tests/tracked_with_struct_db.rs

```diff
@@ -51,16 +51,24 @@ fn execute() {
     let t1 = create_tracked_list(&db, input);
     expect_test::expect![[r#"
         MyTracked {
-            [salsa id]: 1,
+            [salsa id]: Id(
+                1,
+            ),
             data: MyInput {
-                [salsa id]: 0,
+                [salsa id]: Id(
+                    0,
+                ),
```

Comment:
Okay, the fact that `debug_tuple` splits the ID across multiple lines in alternate mode is too verbose. I'm just gonna use a regular write call.